sunflower-land/sunflower-land
# [CHORE] Standardize pixel sizes for all placed collectables

# Description

- standardize pixel size and alignments for all placed collectibles with the exception of Nancy
- better alignments with grid for placed collectibles
- fix Rock Golem kneel down condition
- increase height of Wood Nymph Wendy to 2
- fix tree sprite sheets positions
- fix resource nodes clickable outside of their squares

![image](https://user-images.githubusercontent.com/107602352/206062609-e054cd5c-d7f1-433b-94ea-0f1550f7dd47.png)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

- have all collectables placed in your farm
- hover mouse over resource nodes and see where they are clickable
- use shovels on placed collectables and make sure they are still diggable
- interact with interactable NFTs (eg. Fountain / Wicked Man) and make sure they are still interactable
- mine any stone with Rock Golem built

# Checklist:
